src/App.js

```diff
@@ -1,16 +1,173 @@
 import React from "react";
 import logo from "./logo.png";
 import "./App.css";
+import axios from "axios";
+import "bootstrap/dist/css/bootstrap.min.css";
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      apiKey: "db2b631c79deae1500162a294faec7dc",
+      output: [],
+      cityInput: "",
+      img: "",
+      errorMsg: "",
+    };
+  }
+
+  handleChange = (e) => {
+    this.setState({
+      cityInput: e.target.value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+
+    axios
+      .get(
+        `https://api.openweathermap.org/geo/1.0/direct?q=${this.state.cityInput}&limit=1&appid=db2b631c79deae1500162a294faec7dc`
+      )
+      .then((response) => response.data[0])
+      .then((cityGeoData) =>
+        axios.get(
+          `https://api.openweathermap.org/data/2.5/weather?lat=${cityGeoData.lat}&lon=${cityGeoData.lon}&appid=db2b631c79deae1500162a294faec7dc&units=metric`
+        )
+      )
+      .then((response) => {
+        const { data: weatherData } = response;
+        console.log(weatherData);
+        console.log(weatherData.weather[0].icon);
+        this.setState({
+          output: weatherData,
+        });
+
+        axios
+          .get(
+            `https://openweathermap.org/img/wn/${weatherData.weather[0].icon}@2x.png`
+          )
+          .then((res) => {
+            this.setState({
+              img: res.config.url,
```

Comment:
Does this work? Since you do not return the axios request's result in the previous then block, I do not think this would work.